### ISSUE TITLE
PR workflow: testing

### ISSUE THREAD

**Issue by cellistigs** [cunningham-lab/neurocaas#74]
In the future, it would be great to add additional features: 
- run tests automatically on pull request to make sure nothing fishy is going on (although we'd have to approve tests running on forks somehow too). 
    - This is already done! The pr requires that status checks pass before starting a deployment. Kind of annoying, but okay. 
- have additional tags #develop:{stackname}, #test:{stackname} to deploy a testing version, or even a localstack version before touching production analyses. 
    - you can link these to the iac_utils/build.sh and iac_utils/test_main.sh scripts IF you correctly format and put data into the given buckets. 
- have additional deployment environments, and link those urls to the analyses, or websites in aws. 
- Handle different regions.
